edit.go

```diff
@@ -49,10 +49,16 @@ func simpleEditor(v *View, key Key, ch rune, mod Modifier) bool {
 		v.TextArea.ToggleOverwrite()
 	case key == KeyCtrlU:
 		v.TextArea.DeleteToStartOfLine()
+	case key == KeyCtrlK:
+		v.TextArea.DeleteToEndOfLine()
 	case key == KeyCtrlA || key == KeyHome:
 		v.TextArea.GoToStartOfLine()
 	case key == KeyCtrlE || key == KeyEnd:
 		v.TextArea.GoToEndOfLine()
+	case key == KeyCtrlW:
+		v.TextArea.BackSpaceWord()
+	case key == KeyCtrlY:
```

Comment:
Is there a corresponding paste keybinding for this? I assume the text doesn't go into your actual clipboard